client/state/shortcodes/reducer.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import { LOAD_STATUS } from './constants';
+import { shortcodesSchema } from './schema';
+import {
+	SHORTCODE_FETCH,
+	SHORTCODE_RECEIVE
+} from 'state/action-types';
+
+const siteShortcode = ( state = {}, { type, error, data } ) => {
+	switch ( type ) {
+		case SHORTCODE_FETCH:
+			return { ...state, ...{ status: LOAD_STATUS.LOADING } };
```

Comment:
This expression can be shortened to:

```js
return { ...state, status: LOAD_STATUS.LOADING };
```

Typically you'd only need the spread operator when adding properties from an existing object, but if you're creating a new one, you can just add the properties directly.